src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GenerateVCFFromPosteriors.java

```diff
@@ -0,0 +1,224 @@
+        oneLineSummary = "Create a VCF given the output of gCNV caller",
+        programGroup = CopyNumberProgramGroup.class
+)
+public class GenerateVCFFromPosteriors extends GATKTool {
+    private static final Logger logger = LogManager.getLogger(GenerateVCFFromPosteriors.class);
+
+    public static final String POSTERIOR_CALL_DIRECTORY_FULL_NAME = "called-chunk-directory";
+    public static final String POSTERIOR_CALL_DIRECTORY_SHORT_NAME = "CCD";
+
+    public static final String SAMPLE_DIRECTORY_NAME_FULL_NAME = "sample-name-directory";
+    public static final String SAMPLE_DIRECTORY_NAME_SHORT_NAME = "SN";
+
+    private static final String COMMENT_PREFIX = "@";
+
+    @Argument(
+            doc = "List of paths to chunk directories in a sorted order",
+            fullName = POSTERIOR_CALL_DIRECTORY_FULL_NAME,
+            shortName = POSTERIOR_CALL_DIRECTORY_SHORT_NAME
+    )
+    private List<File> orderedChunkDirectoryList = null;
+
+    @Argument(
+            doc = "Name of the sample directory in the gCNV called chunks (it must be the same across all chunks)",
+            fullName = SAMPLE_DIRECTORY_NAME_FULL_NAME,
+            shortName = SAMPLE_DIRECTORY_NAME_SHORT_NAME
+    )
+    private String sampleDirectoryName = null;
+
+    @Argument(
+            doc = "Output VCF file",
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME
+    )
+    private File outputFile = null;
+
+    /**
+     * Sample name derived from the copy number posterior file
+     */
+    private String sampleName;
+
+    final String variantPrefix = "CNV";
+
+    @Override
+    public void traverse() {
+        if (orderedChunkDirectoryList.size() <= 0) {
+            throw new UserException.BadInput("You must specify at least one chunk directory");
+        }
+
+        final SimpleIntervalCollection firstChunkSimpleIntervalCollection =
+                new SimpleIntervalCollection(new File(getIntervalFileFromChunkDirectory(orderedChunkDirectoryList.get(0).getAbsolutePath()).getAbsolutePath()));
+        final SAMSequenceDictionary samSequenceDictionary = firstChunkSimpleIntervalCollection.getMetadata().getSequenceDictionary();
+        Triple<List<CopyNumberPosteriorLocatableRecord>, IntegerCopyNumberStateCollection, String> firstCopyNumberPosteriorFileInfo =
+                readChunkedPosteriorFileFromDirectory(orderedChunkDirectoryList.get(0), samSequenceDictionary);
+        final IntegerCopyNumberStateCollection copyNumberStateCollection = firstCopyNumberPosteriorFileInfo.getMiddle();
+        sampleName = firstCopyNumberPosteriorFileInfo.getRight();
+
+        if (firstCopyNumberPosteriorFileInfo.getMiddle().size() < 3) {
+            throw new UserException.BadInput("There should be at least 3 copy number states");
+        }
+
+        final VariantContextWriter outputWriter = GATKVariantContextUtils.createVCFWriter(outputFile, null, false);
```

Comment:
@asmirnov239 Since this is a GATKTool, you should be able to use GATKTool.createVCFWriter to create the writer (it will honor all of the other common arguments for index creation, md5, leniency, etc).

Reply:
Ah, OK---perhaps ignore my comments about making this a CLP above then.

Reply:
Agree with @cmnbroad that you should call `GATKTool.createVCFWriter()` here so that your tool will honor the standard GATK arguments for index creation, etc.

Reply:
Done!